### PR TITLE
1.14.3 checkpoints

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -196,16 +196,22 @@ class CMainParams : public CChainParams {
...
+            ( 3606083, uint256S("0x954c7c66dee51f0a3fb1edb26200b735f5275fe54d9505c76ebd2bcabac36f1e"))
         };
 
         chainTxData = ChainTxData{
-            // Data as of block 77e3f4a4bcb4a2c15e8015525e3d15b466f6c022f6ca82698f329edef7d9777e (height 2510150).
-            // Tx estimate based on average of year 2018 (~27k transactions per day)
-            1544484077, // * UNIX timestamp of last checkpoint block
-            42797508,   // * total number of transactions between genesis and last checkpoint
+            // Data as of block 954c7c66dee51f0a3fb1edb26200b735f5275fe54d9505c76ebd2bcabac36f1e (height 3606083).
+            // Tx estimate based on average of year 2021 (~40k transactions per day)
```
Comment (langerhans) [inline src/chainparams.cpp:210]: 2021 has seen some above average activity I'd guess. Are we expecting this trend to hold on for a while longer?

**Review by langerhans — COMMENTED** [review]
Verified the mainnet hashes against my local chain. Do wonder about the tx estimate.

File: src/chainparams.cpp (hunk 1 above)
Comment (rnicoll) [reply src/chainparams.cpp:210]: 2020 doesn't seem much slower, actually. Year start/end looks like:

```
UpdateTip: new best=f6bd77f8c2ba6053bea94120a24f41a8480a38954098af5ab5e0f4d411ccfea8 height=3043797 version=0x00620103 log2_work=73.594646 tx=54490003 date='2020-01-01 00:00:23' progress=0.831138 cache=245.8MiB(268360tx)
UpdateTip: new best=cf8997cac1ad13837a821b2c02fadb07b39500f8f1a13ed01552ac3c0b90c1a3 height=3606004 version=0x00620104 log2_work=74.195891 tx=71033379 date='2021-02-13 10:47:18' progress=1.000000 cache=6.3MiB(1744tx)
```

71,033,379 - 54,490,003 = 16,543,376
16,543,376 / 365 / 24 / 3,600 = 0.5246 tx/s

Thoughts?

Comment (patricklodder) [reply src/chainparams.cpp:210]: I'm good with 0.46 as an estimate.

Comment (langerhans) [reply src/chainparams.cpp:210]: Works for me

**Review by langerhans — APPROVED** [review]

**Comment by langerhans** [comment]
Needs rebase

**Review by patricklodder — APPROVED** [review]

**Pull request merged**